integration/service/oasis/spatialindexer/interface_mock.go

```diff
@@ -1,6 +1,7 @@
 package spatialindexer
 
-var mockPlaceInfo1 = []*PointInfo{
+// MockPlaceInfo1 contains 10 PointInfo items
+var MockPlaceInfo1 = []*PointInfo{
```

Comment:
Why export the mock var?

Reply:
I want to use which to evaluate result
```
package A defines some common data structures

package B use that common data structures and returns some result, I use package A's mock result to evaluate whether result is the same as expecdted.

```

Reply:
Then the `B` will strongly depends on `A`, which may should not. Maybe you could write some `test` packages to test the scenario like this?